test/gtest/uct/test_uct_perf.cc

```diff
@@ -25,77 +25,77 @@ test_perf::test_spec test_uct_perf::tests[] =
 {
   { "am latency", "usec",
     UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_PINGPONG,
-    UCT_PERF_DATA_LAYOUT_SHORT, 8, 1, 100000l,
+    UCT_PERF_DATA_LAYOUT_SHORT, 0, 1, { 8 }, 1, 100000l,
     ucs_offsetof(ucx_perf_result_t, latency.total_average), 1e6, 0.01, 2.5},
 
   { "am rate", "Mpps",
     UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_STREAM_UNI,
-    UCT_PERF_DATA_LAYOUT_SHORT, 8, 1, 2000000l,
+    UCT_PERF_DATA_LAYOUT_SHORT, 0, 1, { 8 }, 1, 2000000l,
     ucs_offsetof(ucx_perf_result_t, msgrate.total_average), 1e-6, 0.8, 80.0 },
 
   { "am rate64", "Mpps",
     UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_STREAM_UNI,
-    UCT_PERF_DATA_LAYOUT_SHORT, 64, 1, 2000000l,
+    UCT_PERF_DATA_LAYOUT_SHORT, 0, 1, { 64 }, 1, 2000000l,
     ucs_offsetof(ucx_perf_result_t, msgrate.total_average), 1e-6, 0.8, 80.0 },
 
   { "am bcopy bw", "MB/sec",
     UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_STREAM_UNI,
-    UCT_PERF_DATA_LAYOUT_BCOPY, 1000, 1, 100000l,
+    UCT_PERF_DATA_LAYOUT_BCOPY, 0, 1, { 1000 }, 1, 100000l,
     ucs_offsetof(ucx_perf_result_t, bandwidth.total_average), MB, 620.0, 15000.0 },
 
   { "am zcopy bw", "MB/sec",
     UCX_PERF_API_UCT, UCX_PERF_CMD_AM, UCX_PERF_TEST_TYPE_STREAM_UNI,
-    UCT_PERF_DATA_LAYOUT_ZCOPY, 1000, 32, 100000l,
+    UCT_PERF_DATA_LAYOUT_ZCOPY, 8192, 3, { 256, 256, 512 }, 32, 100000l,
```

Comment:
would be better to create another test (e.g "am iov bw") instead of changing existing one, in case the transport would not support such amount of IOVs